src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -42,14 +43,16 @@ module.exports = class ArtworkCommercialView extends Backbone.View
     # Show the new buy now flow if you have the lab feature enabled
     loggedInUser = CurrentUser.orNull()
     if loggedInUser?.hasLabFeature('New Buy Now Flow')
+      serializer = new Serializer @$('form')
+      data = serializer.data()
+
       createOrder
-        user: loggedInUser
-        partnerId: @artwork.get('partner_id')
-        currencyCode: "usd"
         artworkId: @artwork.get('_id')
-        priceCents: 500000, quantity: 1
+        editionSetId: data.edition_set_id
+        quantity: 1
```

Comment:
Is the quantity a hard limit or is this just a placeholder value?

Reply:
It actually defaults to `1` in exchange but I thought it best to be explicit. In the future we'll likely want to support being able to buy multiples of the same work but there are no plans/UI for that now so it will always be 1.